### PR TITLE
feat: add snake to graphql and graphql to snake functions

### PR DESCRIPTION
**Pull request by thomas9911**

### CODE REVIEW

File: lib/case_style.ex [hunk 1]
```diff
@@ -12,13 +12,11 @@ defmodule CaseStyle do
...
-          {:error, any, any, any, any, any}
-  # {:error, list, binary, map, {pos_integer, pos_integer}, pos_integer}
+          {:error, binary, binary, map, {pos_integer, pos_integer}, pos_integer}
+  @type parser_output :: {:ok, list, binary, map, {pos_integer, pos_integer}, pos_integer}
 
-  @type parser_ouput :: {:ok, list, binary, map, {pos_integer, pos_integer}, pos_integer}
-
-  # @callback parse(input :: binary) :: {:ok, CaseStyle.t()} | parser_ouput | nimble_parsec_error
-  @callback parse(input :: binary) :: parser_ouput | nimble_parsec_error
+  # @callback parse(input :: binary) :: {:ok, CaseStyle.t()} | parser_output | nimble_parsec_error
```
Comment (rodrigo-dap) [inline lib/case_style.ex:18]: rm comments?

File: lib/case_style.ex [hunk 2]
```diff
@@ -12,13 +12,11 @@ defmodule CaseStyle do
...
+  @callback parse(input :: binary) :: parser_output | nimble_parsec_error
   @callback might_be?(input :: binary) :: boolean
```
Comment (rodrigo-dap) [inline lib/case_style.ex:20]: might_be? the suspense is killing me :joy:

File: lib/case_style/graphql_case.ex [hunk 3]
```diff
@@ -0,0 +1,37 @@
+defmodule CaseStyle.GraphQLCase do
+  @moduledoc """
+  Module for converting from and to graphql.
+  This is just camelCase only that it doesnt allow that the string starts with a number.
```
Comment (rodrigo-dap) [inline lib/case_style/graphql_case.ex:4]: can you rephrase this?

File: lib/case_style.ex (hunk 2 above)
Comment (thomas9911) [reply lib/case_style.ex:20]: would have been, should have been, could have been. YOU

**Review by rodrigo-dap — APPROVED** [review]

**Review by rodrigo-dap — APPROVED** [review]
